### PR TITLE
ENTESB-11790: pick latest s2i image for building

### PR DESCRIPTION
**Pull request by rareddy**

### CODE REVIEW

**Comment by heiko-braun** [comment]
@lgarciaaco Mind a review?

**Comment by asmigala** [comment]
Can someone please re-run the ci checks? I don't have the permission to do that and there seems to be no reason for the failures other than some timeouts

File: install/operator/pkg/generator/assets/addons/komodo/addon-komodo-server.yml.tmpl [hunk 1]
```diff
@@ -61,7 +61,7 @@
           - name: GC_MAX_METASPACE_SIZE
             value: "512"
           - name: BUILD_IMAGE_STREAM
-            value: {{ .Images.Syndesis.S2i }}:latest
+            value: {{ .Images.Syndesis.S2i }}:{{.Syndesis.Spec.Components.S2I.Image}}
```
Comment (asmigala) [inline install/operator/pkg/generator/assets/addons/komodo/addon-komodo-server.yml.tmpl]:
```suggestion
            value: {{ .Images.Syndesis.S2i }}:{{.Syndesis.Spec.Components.S2I.Tag}}
```

**Comment by asmigala** [comment]
Turns out there _was_ a reason for a failure

**Comment by rareddy** [comment]
I will update the PR

**Pull request merged**